datafusion/core/src/physical_plan/windows/bounded_window_agg_exec.rs

```diff
@@ -73,6 +75,8 @@ pub struct BoundedWindowAggExec {
     pub partition_keys: Vec<Arc<dyn PhysicalExpr>>,
     /// Execution metrics
     metrics: ExecutionPlanMetricsSet,
+    /// Partition by indices that define ordering
```

Comment:
I am still a little confused about what a partition by index that defines an ordering and one that does not define an ordering means. Maybe we could change the text to be more explicit 

For example, is this a subset of the partition by expressions that are also part of hte ORDER by clause

Maybe we could offer an example of what indexes this set would be for a query like

```
OVER (PARTITION BY a, b ORDER BY y, z)
```

Reply:
Maybe putting this documentation on 

```
pub(crate) fn get_ordered_partition_by_indices(
``` 

and saying "see `get_ordered_partition_by_indices` for more details"  would be a good structure

Reply:
I added documentation on `get_ordered_partition_by_indices`. Updated comment here to refer documentation for more info. Thanks for suggestion.